demo/app/lib/wallet_sdk/wallet_sdk_mobile.dart

```diff
@@ -43,6 +43,17 @@ class WalletSDK extends WalletPlatform {
     }
   }
 
+  Future<Map<Object?, Object?>?> initializeWalletInitiatedFlow(Map<String, dynamic>? walletInitiateFlowArgs) async {
```

Comment:
Return some typed structure instead of Map<Object?, Object?>?

See as example https://github.com/trustbloc/wallet-sdk/blob/bb013ae31095234ed3a45c703b1f8af5e02d68ae/demo/app/lib/wallet_sdk/wallet_sdk_mobile.dart#L94